vm/src/builtins.rs

```diff
@@ -793,7 +795,7 @@ pub fn make_module(ctx: &PyContext) -> PyObjectRef {
         "repr" => ctx.new_rustfunc(builtin_repr),
         "reversed" => ctx.new_rustfunc(builtin_reversed),
         "round" => ctx.new_rustfunc(builtin_round),
-        "set" => ctx.set_type(),
+        "set" => objset::PySet::class_from_ctx(ctx),
```

Comment:
I would prefer here: `vm.types.set_type()`. I think it is okay to have a collection of builtin types stored in some struct, like the exceptions.